manifests/init.pp

```diff
@@ -421,6 +422,10 @@
     fail('ssh::sshd_config_banner must be set to be able to use sshd_banner_content.')
   }
 
+  if $ssh_gssapiauthentication != undef {
```

Comment:
Good idea. Since that would be a bugfix and your are adding a new feature, it should go into a separate branch and pull request.